SPECS-EXTENDED/proj/proj.spec

```diff
@@ -0,0 +1,959 @@
+data with a wide range of selectable projection functions.
+
+%package -n %{libname}
+Summary:        Cartographic projection software
+License:        MIT
+
+%description -n %{libname}
+This package the library for performing respective
+forward and inverse transformation of cartographic data to or from cartesian
+data with a wide range of selectable projection functions.
+
+%package devel
+Summary:        Development files for PROJ
+License:        MIT
+Requires:       %{libname} = %{version}
+Provides:       libproj-devel = %{version}
+Obsoletes:      libproj-devel < %{version}
+
+%description devel
+This package contains libproj and the appropriate header files and man pages.
+
+%define data_subpkg(c:n:e:s:) \
+%define countrycode %{-c:%{-c*}}%{!-c:%{error:Country code not defined}} \
+%define countryname %{-n:%{-n*}}%{!-n:%{error:Country name not defined}} \
+%define extrafile %{-e:%{_datadir}/%{name}/%{-e*}} \
+%define wildcard %{!-s:%{_datadir}/%{name}/%{countrycode}_*} \
+\
+%package data-%{countrycode}\
+Summary:      %{countryname} datum grids for Proj\
+BuildArch:    noarch\
+# See README.DATA \
+License:      BSD-2-Clause AND CC0-1.0 AND CC-BY-4.0 AND CC-BY-SA-4.0 AND SUSE-Public-Domain \
+Supplements:  proj\
+\
+%description data-%{countrycode}\
+%{countryname} datum grids for Proj.\
+\
+%files data-%{countrycode}\
+%{wildcard}\
+%{extrafile}
+License:        MIT
+%data_subpkg -c at -n Austria
+%data_subpkg -c au -n Australia
+%data_subpkg -c be -n Belgium
+%data_subpkg -c br -n Brasil
+%data_subpkg -c ca -n Canada
+%data_subpkg -c ch -n Switzerland
+%data_subpkg -c de -n Germany
+%data_subpkg -c dk -n Denmark -e DK
+%data_subpkg -c es -n Spain
+%data_subpkg -c eur -n %{quote:Nordic + Baltic} -e NKG
+%data_subpkg -c fi -n Finland
+%data_subpkg -c fo -n %{quote:Faroe Island} -e FO -s 1
+%data_subpkg -c fr -n France
+%data_subpkg -c is -n Island -e ISL
+%data_subpkg -c jp -n Japan
+%data_subpkg -c mx -n Mexico
+%data_subpkg -c nc -n %{quote:New Caledonia}
+%data_subpkg -c nl -n Netherlands
+%data_subpkg -c no -n Norway
+%data_subpkg -c nz -n %{quote:New Zealand}
+%data_subpkg -c pl -n Poland
+%data_subpkg -c pt -n Portugal
+%data_subpkg -c se -n Sweden
+%data_subpkg -c sk -n Slovakia
+%data_subpkg -c uk -n %{quote:United Kingdom}
+%data_subpkg -c us -n %{quote:United States}
+%data_subpkg -c za -n %{quote:South Africa}
+
+%prep
+%autosetup
+
+%build
+%cmake -DCMAKE_INSTALL_MANDIR:PATH=%{_mandir}
+%cmake_build
+
+%install
+%cmake_install
+tar -C %{buildroot}%{_datadir}/%{name} -xf %{SOURCE1}
+find %{buildroot} -type f -name "*.la" -delete -print
+# It would be good to find out where these extra files
+# come from:
+rm -rf %{buildroot}%{_datadir}/doc/${name}
+
+%check
+# Tests dont work on i586 and noone cares
+%ifnarch %{ix86}
+%ctest
+%endif
+
+%post -n %{libname} -p /sbin/ldconfig
+%postun -n %{libname} -p /sbin/ldconfig
+
+%files
+%doc NEWS AUTHORS README ChangeLog
```

Comment:
Maybe add a `license` tag in the main rpm aswell.

Reply:
Done